translations/recommendations.tex/pl.po

```diff
@@ -175,16 +277,38 @@ msgstr ""
 "Mimo że zasady mówią, że możesz usunąć tylko jedną kartę z ręki podczas odwiedzania Punktu Handlowego, zaleca się pozwolić na usunięcie tylu kart, ile chcesz.\n"
```

Comment:
```suggestion
"Mimo że zasady mówią, że możesz usunąć tylko jedną kartę z ręki podczas odwiedzania Punktu Handlowego, zaleca się pozwolić na usunięcie dowolnej ilości kart.\n"
```

Reply:
Liczby zamiast ilości, ale tak to poprawka ok.